addon/webextension/selector/ui.js

```diff
@@ -91,6 +91,31 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
     return result;
   }
 
+  function isDownloadOnly() {
+    return window.incognito;
+  }
+
+  function renderDownloadNotice() {
+    let notice = makeEl("div", "download-only");
+    notice.textContent = browser.i18n.getMessage("downloadOnlyNotice");
+    let questionMark = makeEl("span", "circle");
+    questionMark.textContent = "?";
+    notice.appendChild(questionMark);
+
+    let tooltip = makeEl("div", "download-only-details");
+    let details = makeEl("p");
+    details.textContent = browser.i18n.getMessage("downloadOnlyDetails");
+    let detailsList = makeEl("ul");
+    let detailsPrivate = makeEl("li");
+    detailsPrivate.textContent = browser.i18n.getMessage("downloadOnlyDetailsPrivate");
+
+    detailsList.appendChild(detailsPrivate);
+    details.appendChild(detailsList);
+    tooltip.appendChild(details);
+    notice.appendChild(tooltip);
+    return notice;
```

Comment:
Isn't there a function that can localize this?  I.e., we do:

```js
let notice = `<div class="download-only">
<p><span data-l10n-id="downloadOnlyNotice"></span><span class="circle">?</span>
...
`;
```

That's how we setup the iframes I think.